charts/karmada/_crds/bases/autoscaling.karmada.io_federatedhpas.yaml

```diff
@@ -19,7 +19,26 @@ spec:
     singular: federatedhpa
   scope: Namespaced
   versions:
-  - name: v1alpha1
+  - additionalPrinterColumns:
+    - jsonPath: .spec.scaleTargetRef.kind
+      name: REFERENCEKIND
+      type: string
+    - jsonPath: .spec.scaleTargetRef.name
+      name: REFERENCENAME
+      type: string
+    - jsonPath: .spec.minReplicas
+      name: MINPODS
+      type: integer
+    - jsonPath: .spec.maxReplicas
+      name: MAXPODS
+      type: integer
+    - jsonPath: .status.currentReplicas
+      name: REPLICAS
```

Comment:
How about target and current utilization?

Reply:
I am afraid that kubebuilder cannot do it. Because for metrics spec, the json path is not definite. E.g. for resources of cpu, it may be metrics[0].resource.target.averageUtilization.